src/PokemonGymImpl.java

```diff
@@ -144,7 +137,7 @@ public String chooseAttackPlayer(Pokemon p){
     }
 
     @Override
-    public void performAttackPlayer(Pokemon pokemon, Pokemon gymPokemon, String attack){
+    public void performAttackPlayer(Pokemon pokemon, Pokemon gymPokemon, String attack) {
```

Comment:
Het is goed dat je al een stijl mening gevormd hebt dat je een spatie wilt tussen ) en { 

functioneel maakt het niks uit of je er een spatie tussen zet of niet, voor de leesbaarheid ook niet, maar goed dat je er wel consequent in bent.